repositories.yaml

```diff
@@ -243,6 +257,7 @@ applications:
       # does not actually depend on glean-android, including for backwards
       # compat
       - org.mozilla.components:service-glean
+      - nimbus
```

Comment:
```suggestion
      - org.mozilla.components:nimbus
```

Reply:
This is iOS, does it need the android library dependency listed here like Glean does?